examples/README.md

```diff
@@ -38,6 +38,48 @@ This dashboard shows, how far we have checked the examples in the Quint-Apalache
 pipeline. The reported status reflects running the noted subcommand on the
 listed without any additional command line arguments.
 
+| Example | Syntax (`parse`) | Types (`typecheck`) | Unit tests (`test`) | Apalache (`verify`) |
+|---------|:----------------:|:-------------------:|:-------------------:|:-------------------:|
+<!-- DO NOT EDIT: THIS FILE IS GENERATED FROM examples/.scripts/README-text.md VIA 'make examples' -->
+This directory contains examples of protocol specifications written in Quint.
+
+Note that some examples were written long time ago. All of our examples pass
+the parser and the type checker, but some of them are not runnable and testable
+yet. To set your expectations right, check the dashboard below first.
+
+# Categories
+
+ - [solidity](./solidity). The most recent examples of Solidity smart contracts
```

Comment:
Looks like something has gone wrong here: the full output is appended to template header...

Reply:
Yeah, I'm waiting for https://github.com/informalsystems/quint/pull/989 to get merged, then I'll run a fresh make examples on this branch

Reply:
Looks like the `verify` checks are failing https://github.com/informalsystems/quint/actions/runs/5400704233/jobs/9809505038?pr=985#step:10:42

Try running `make apalache`, and see if that sets things straight. I'll think where to document this too.

Reply:
Looks mostly better, but `Coin.qnt` still doesn't work

Reply:
You probably have to recompile `quint` now that you are up to date with main